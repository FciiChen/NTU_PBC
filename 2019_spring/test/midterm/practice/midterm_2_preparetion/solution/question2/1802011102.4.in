6So, HAUfTg6,US0UUjSk
3MxQsOnq, fmaWZa8JV,tK5Rpo0vA
Y1tNmA, BNZDV,rKj
Ox4D6, vsHSum,Er9zT7w
BREAK
Y1tNmA ,  bonQ4K
Ox4D6 ,  yMCt
6So ,  zD2scsXuG
3MxQsOnq ,  xqc4
BREAK
