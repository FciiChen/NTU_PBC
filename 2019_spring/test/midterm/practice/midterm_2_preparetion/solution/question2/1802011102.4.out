6So        HAUfTg6    US0UUjSk   zD2scsXuG 
3MxQsOnq   fmaWZa8JV  tK5Rpo0vA  xqc4      
Y1tNmA     BNZDV      rKj        bonQ4K    
Ox4D6      vsHSum     Er9zT7w    yMCt      
